libs/execution_context/ebpf_native.c

```diff
@@ -309,6 +311,7 @@ _Requires_exclusive_lock_held_(module->lock) static void _ebpf_native_acquire_re
 void
 ebpf_native_acquire_reference(_Inout_ ebpf_native_module_t* module)
 {
+    EBPF_LOG_ENTRY();
```

Comment:
I believe this macro must always be paired with another macro.
In this case, I think it's `EBPF_RETURN_VOID();` at the end.

But I think this is a hot path.  Do you really want to be introducing log calls into the hot path?  How much of a perf hit is it?

Reply:
This is not in the hot path. Native module reference is acquired only when the native module is loaded, and released when the ebpf programs and/or the native module are unloaded.

But agree about pairing. I will change the tracing.

Reply:
Actually ended up removing this, as there is a reference history also maintained.